3-enrich/scala-hadoop-shred/src/main/scala/com.snowplowanalytics.snowplow.enrich/hadoop/ShredJob.scala

```diff
@@ -146,7 +146,7 @@ class ShredJob(args : Args) extends Job(args) {
       ShredJob.projectBads(o)
     }
     .mapTo(('line, 'errors) -> 'json) { both: (String, ProcessingMessageNel) =>
-      BadRow(both._1, both._2).toCompactJson
+      new BadRow(both._1, both._2).toCompactJson
```

Comment:
Without the `new` the compiler interprets that we want to use [BadRow#apply](https://github.com/snowplow/snowplow/blob/feature/r71/3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/outputs/BadRow.scala#L48), which uses Strings instead of ProcessingMessages.

I just added the new because it was the smallest change to make the code compile. Otherwise I could create a new `BadRow#apply` that uses ProcessingMessages or I could `both._2.map(_.toString)`.